### PR TITLE
Change the command in test_pids_max function into bash script

### PR DESCRIPTION
**Pull request by badboywj**
the [issue](https://openqa.suse.de/tests/14367747#step/hana_test/23) happened frequently because of worker performance issue, the command input is error, and running bash script is more stable than running command in terminal, this PR finished the change

- Related ticket: https://jira.suse.com/browse/TEAM-9362
- Needles: None
- Verification run:
    https://openqa.suse.de/tests/14485767
    https://openqa.suse.de/tests/14476230
    https://openqa.suse.de/tests/14485784

### CODE REVIEW

**Review by lilyeyes — APPROVED** [review]
LGTM.

**Review by Amrysliu — APPROVED** [review]
LGTM

File: lib/sles4sap.pm [hunk 1]
```diff
@@ -517,14 +517,30 @@ and threads that it can create.
 sub test_pids_max {
     # UserTasksMax should be set to "infinity" in /etc/systemd/logind.conf.d/sap.conf
     my $uid = script_output "id -u $sapadmin";
+
     # The systemd-run command generates syslog output that may end up in the console, so save the output to a file
```
Comment (mpagot) [inline lib/sles4sap.pm:521]: evaluate to update this comment to explain why the execution is within a `sh` script

Comment (badboywj) [reply lib/sles4sap.pm:521]: **systemd-run** command string is too long, it is not stable to type the command in terminal and failed to execute the command, it is more stable to execute the command in script

Comment (badboywj) [reply lib/sles4sap.pm]: I have updated the comment

**Review by mpagot — APPROVED** [review]
LGTM

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Comment by alvarocarvajald** [comment]
We need to address the softfail in 15-SP6 (on an upcoming PR, not this one).

I think issue comes from `/sys/fs/cgroup/pids/user.slice/user-1001.slice/pids.max` no longer being a valid path in cgroups v2 which comes on 15-SP6. See: https://openqa.suse.de/tests/14485767#step/hana_test/15

A quick look into an x86_64 VM created from 15-SP6 GMC ISO makes me think the new path is: `/sys/fs/cgroup/user.slice/user-1001.slice/pids.max`. I'll create a ticket to address this

**Pull request merged**